rgsync/redis_gears_write_behind.py

```diff
@@ -381,11 +384,12 @@ def __init__(self, GB, keysPrefix, mappings, connector, name, version=None,
             'desc':'add each changed key with prefix %s:* to Stream' % keysPrefix,
         }
         GB('KeysReader', desc=json.dumps(descJson)).\
+        map(transform).\
         filter(ValidateHash).\
         filter(ShouldProcessHash).\
         foreach(DeleteHashIfNeeded).\
         foreach(CreateAddToStreamFunction(self)).\
-        register(mode='sync', prefix='%s:*' % keysPrefix, eventTypes=['hset', 'hmset', 'del', 'change'])
+        register(mode='sync', regex='%s:*' % keysPrefix, eventTypes=eventTypes)
```

Comment:
regex is deprecated, you are probably still running on 0.9. Please change to prefix